examples/mapviewer.py

```diff
@@ -189,29 +208,26 @@ def load_map_clicked(self, button):
         format = self.image_format_entry.get_text()
         if uri and format:
             if self.osm:
-                #remove old map
+                # remove old map
                 self.vbox.remove(self.osm)
             try:
-                self.osm = osmgpsmap.Map(
-                    repo_uri=uri,
-                    image_format=format
-                )
-            except Exception, e:
-                print "ERROR:", e
-                self.osm = osm.Map()
+                self.osm = osmgpsmap.Map(repo_uri=uri, image_format=format)
+            except Exception as e:
+                print("ERROR:", e)
```

Comment:
Just for the record: Writing the `print` statements like this isn't backwards-compatible with Python 2.

Python 2:
```
>>> print("foo", "bar")
('foo', 'bar')
```
Python 3:
```
>>> print("foo", "bar")
foo bar
```
It writes out with Python 3, though, what was written out with the old code on Python 2:
```
>>> print "foo", "bar"
foo bar
```
Let's agree that Python 2 is meant to be dead for this example. :checkered_flag:

Reply:
... the f-strings below require Python >= 3.6 anyway. True.